buildlib/pr/main.yml

```diff
@@ -0,0 +1,393 @@
+            env:
+              LD_LIBRARY_PATH: $(System.DefaultWorkingDirectory)/ucx-1.11/install/lib
+            displayName: Test ucx-1.11 version
+          - bash: |
+              set -Eex
+              LD_LIBRARY_PATH=$UCX_V11_LIB_PATH $(System.DefaultWorkingDirectory)/ucx-1.11/examples/ucp_client_server -c tag &
+              server_pid=$!
+              sleep 5
+              LD_LIBRARY_PATH=$UCX_PR_LIB_PATH $(System.DefaultWorkingDirectory)/ucx-1.11/examples/ucp_client_server -c tag -a `hostname -I`
+              kill -9 $server_pid
+            displayName: Test wire compatibility server v1.11 client master
+            env:
+              UCX_V11_LIB_PATH: $(System.DefaultWorkingDirectory)/ucx-1.11/install/lib
+              UCX_PR_LIB_PATH: $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId)/install/lib
+          - bash: |
+              set -Eex
+              LD_LIBRARY_PATH=$UCX_PR_LIB_PATH $(System.DefaultWorkingDirectory)/ucx-1.11/examples/ucp_client_server -c tag &
+              server_pid=$!
+              sleep 5
+              LD_LIBRARY_PATH=$UCX_V11_LIB_PATH $(System.DefaultWorkingDirectory)/ucx-1.11/examples/ucp_client_server -c tag -a `hostname -I`
+              kill -9 $server_pid
+            env:
+              UCX_V11_LIB_PATH: $(System.DefaultWorkingDirectory)/ucx-1.11/install/lib
+              UCX_PR_LIB_PATH: $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId)/install/lib
+            displayName: Test wire compatibility server on master client v1.11
+
+  - stage: Coverity
+    dependsOn: [Static_check]
+    jobs:
+      - template: coverity.yml
+        parameters:
+          demands: ucx_docker -equals yes
+          container: coverity_rh7
+
+  - stage: Tests
+    dependsOn: [Static_check]
+    jobs:
+    - template: tests.yml
+      parameters:
+        name: althca
+        demands: ucx_althca -equals yes
+        test_perf: 0
+    - template: tests.yml
+      parameters:
+        name: gpu
+        demands: ucx_gpu -equals yes
+        test_perf: 1
+    - template: tests.yml
+      parameters:
+        name: new
+        demands: ucx_new -equals yes
+        test_perf: 1
+    - template: tests.yml
+      parameters:
+        name: hwi
+        demands: ucx_hwi -equals yes
+        test_perf: 0
+    - template: tests.yml
+      parameters:
+        name: sputnik
+        demands: ucx_sputnik -equals yes
+        test_perf: 0
+    - template: tests.yml
+      parameters:
+        name: docker
+        demands:
+          - ucx_docker -equals yes
+          - ucx_amd -equals no
+        container: centos7
+
+  - stage: io_demo
+    dependsOn: [Static_check]
+    jobs:
+    - template: io_demo/io-demo.yml
+
+  - stage: jucx
+    dependsOn: [Static_check]
+    jobs:
+      - template: ../jucx/jucx-test.yml
+        parameters:
+          name: new
+          demands: ucx_new -equals yes
+      - template: ../jucx/jucx-test.yml
```

Comment:
move jucx and go under pr/ ?

Reply:
we are using jucx in snapshot and release

Reply:
ok